lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1837,6 +1837,22 @@ SwiftASTContext::CreateInstance(lldb::LanguageType language, Module &module,
     return {};
   }
 
+  if (swift_ast_sp->HasFatalErrors()) {
+    logError(swift_ast_sp->GetFatalErrors().AsCString());
+    return {};
+  }
```

Comment:
is there something in particular that would cause the fatal errors, separate from the call to `GetClangImporter` failing?

Reply:
GetClangImporter() could in theory create a fatal diagnostic while still succeeding (even though it doesn't do that now, a future version might), so I thought it would be good practice to be in a well-defined state before attempting to import the stdlib.

Reply:
sounds good to me